### PR TITLE
ticdc: add the Storage Sink feature to ticdc-overview.md

### LINKED ISSUE #14957: TiCDC Overview does not provide details about the Storage Sink capabilities
File: [/release-7.1/ticdc/ticdc-overview.md](https://docs.pingcap.com/tidb/stable/ticdc-overview)

This document includes mentions of the TiCDC support for multiple sinks including other TiDB Clusters, MySQL, and Kafka but it does not mentioned TiCDC's support for storage services like AWS S3.

There is some information in other pages of the documentation, such as https://docs.pingcap.com/tidb/stable/ticdc-sink-to-cloud-storage, but this info should also be reflected in the overview page.

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Close #14957 

TiCDC v6.5.0 supports replicating changed logs to Amazon S3, Azure Blob Storage, NFS, and other S3-compatible storage services.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@benmeadowcroft PTAL

**Comment by benmeadowcroft** [comment]
The change you made looks good, but I would also suggest also updating the image on that page as well to include Object storage as a target. Perhaps adding the attached image to your PR and updating the reference to it in the doc
cdc-architecture

**Comment by benmeadowcroft** [comment]
Looks like the image I attached got left off my prior comment somehow, trying again!
cdc-architecture

**Comment by Oreoxmt** [comment]
@benmeadowcroft I have updated the architecture and the description in [TiDB Migration Tools Overview](https://docs.pingcap.com/tidb/stable/migration-tools#ticdc), could you please help review these changes (https://github.com/pingcap/docs/pull/14958/commits/387df20569c572817d24cce0e39df7cb86071d5b)? Thanks!

**Comment by benmeadowcroft** [comment]
@Oreoxmt yes, this PR addresses the issues I raised in https://github.com/pingcap/docs/issues/14957
Thank you!

**Review by hfxsd — APPROVED** [review]

File: ticdc/ticdc-overview.md [hunk 1]
```diff
@@ -70,7 +71,7 @@ The components in the architecture diagram are described as follows:
 - TiCDC: TiCDC nodes where TiCDC processes run. Each node runs a TiCDC process. Each process pulls data changes from one or more tables in TiKV nodes and replicates the changes to the downstream system through the sink component.
 - PD: The scheduling module in a TiDB cluster. This module is responsible for scheduling cluster data and usually consists of three PD nodes. PD provides high availability through the etcd cluster. In the etcd cluster, TiCDC stores its metadata, such as node status information and changefeed configurations.
 
-As shown in the architecture diagram, TiCDC supports replicating data to TiDB, MySQL, and Kafka databases.
+As shown in the architecture diagram, TiCDC supports replicating data to TiDB, MySQL, Kafka, MQ, and storage services.
```
Comment (Oreoxmt) [inline ticdc/ticdc-overview.md:74]:
```suggestion
As shown in the architecture diagram, TiCDC supports replicating data to TiDB, MySQL, Kafka, and storage services.
```

**Pull request merged**